docs/helper-api.md

```diff
@@ -6,25 +6,50 @@ permalink: /docs/helper-api/
 
 ##Dust Helpers
 ###Function definition
-
+```
     dust.helpers.myHelper = function(chunk, context, bodies, params) {
        /* Crazy logic here */
     }
-
+```
 ###Parameters
-* chunk:  the currently accumulating output of the template render process. You will most likely contribute additional output as part of your helper.
-* context: the current context stack (e.g that which changes when you do things like `{#list}`
-* bodies: holds any body sections nested within the helper. For example, the `{:else}`) body.
-* params is an object that holds all the parameters used when calling the custom helper.
-* @return: {Void}
+* chunk: The currently accumulating output of the template render process.
```

Comment:
This doesn't make sense. What is the data type? This should probably be a reference to the full chunk documentation, wherever that will be.